src/jwt.ts

```diff
@@ -25,10 +25,18 @@ const ORCHESTRATOR_KEY = jose.createRemoteJWKSet(
   new URL(process.env.ORCHESTRATOR_PUBLIC_KEY || "https://orchestrator.grindery.org/oauth/jwks")
 );
 
+/**
+ * Parses a user access token and extracts the payload information.
+ *
+ * @param {string} token - The user access token to be parsed.
+ * @returns {Promise<TAccessToken>} A Promise that resolves to the payload of the parsed token.
+ * @throws {Error} If the token verification fails or has invalid format.
+ */
 export async function parseUserAccessToken(token: string): Promise<TAccessToken> {
-  const { payload } = await jose.jwtVerify(token, ORCHESTRATOR_KEY, {
-    issuer: "urn:grindery:orchestrator",
-    audience: "urn:grindery:access-token:v1",
-  });
-  return payload;
+  return (
+    await jose.jwtVerify(token, ORCHESTRATOR_KEY, {
+      issuer: "urn:grindery:orchestrator",
+      audience: "urn:grindery:access-token:v1",
+    })
+  ).payload;
```

Comment:
Don't really like this, makes the code harder to read IMO.

Reply:
@SAPikachu No problem it was to avoid too many lines but I just pushed the old version :)